tracardi/process_engine/action/v1/converters/base64/encode/plugin.py

```diff
@@ -0,0 +1,78 @@
+from base64 import b64encode
+
+from tracardi.service.plugin.domain.register import Documentation, Form, FormComponent, FormField, FormGroup, \
+    MetaData, Plugin, PortDoc, Spec
+from tracardi.service.plugin.domain.result import Result
+from tracardi.service.plugin.runner import ActionRunner
+from .model.models import Configuration
+
+
+def validate(config: dict):
+    return Configuration(**config)
+
+
+class Base64EncodeAction(ActionRunner):
+    config: Configuration
+
+    async def set_up(self, init):
+        self.config = validate(init)
+
+    async def run(self, payload: dict, in_edge=None):
+        dot = self._get_dot_accessor(payload)
+        text = dot[self.config.to_base64]
+        result = b64encode(text.encode(self.config.input_encoding)).decode('ascii')
+
+        return Result(port="payload", value={"base64": result})
+
+
+def register() -> Plugin:
+    return Plugin(
+        start=False,
+        spec=Spec(
+            module=__name__,
+            className=Base64EncodeAction.__name__,
```

Comment:
All other plugins use the hardcoded class name here. Is there a reason for this? i.e.

```
className='Base64EncodeAction'
```

instead of

```
className=Base64EncodeAction.__name__
```

I have no preference and will be happy to conform to the project standard.

The docs use `className=Base64EncodeAction` which will fail the pydantic validator. I will provide a PR to fix the docs after it is clear which form is preferable.

Reply:
Alright, in that case I'll prepare an MR to fix the confusing/misleading/incorrect(?) docs.